src/App.tsx

```diff
@@ -1,24 +1,250 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Filter, Todo, ErrorMessageEnum } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorBin } from './components/ErrorBin/ErrorBin';
+import {
+  addTodo, deleteTodo, getTodos, patchTodo,
+} from './api/todos';
+import { handleError } from './components/ErrorBin/handleError';
 
-const USER_ID = 0;
+const USER_ID = 11572;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [refreshTodos, setRefreshTodos] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [newTodoTitle, setNewTodoTitle] = useState<string>('');
+  const [filter, setFilter] = useState<Filter>('All');
```

Comment:
I would go here with the same exact way as you do with `ErrorMessageEnum`. Create an `const enum` with possible options and this way it's more organized and you don't have to compare to string.

Reply:
Like that 
```
const enum Filter {
    ALL = 'All',
    ACTIVE = 'Active',
    COMPLETED = 'Completed'
}
```

Use it as a type and you can acess this enum member by using `Filter.ALL` etc.